### PR TITLE
fix: Reset client buffer size

### PR DESCRIPTION
**Pull request by RileyYe**
I reset it to 8k, because larger buffer size would ease DDoS attack vectors.

### CODE REVIEW

**Comment by PeterlitsZo** [comment]
Is the 2nd commit is to refactor?

Never mind, I think it is good.

**Review by PeterlitsZo — APPROVED** [review]

**Comment by PeterlitsZo** [comment]
This PR is for large request (for example, a json with a large image encoded in base64).

**Comment by RileyYe** [comment]
Well, I just looked up the dictionary, realizing using `refactor` maybe too bluff...

**Comment by PeterlitsZo** [comment]
Do you think we need rebase your commits to `master` or push right commit message with `--force`?

**Comment by RileyYe** [comment]
I don't mind the both, your call.

**Comment by PeterlitsZo** [comment]
Done. Thanks for your work very much.

By the way, I hope you can follow the git commit message style in `docs/contribute.md`!

If you do not like that style, we can talk about it and make it better.

**Comment by PeterlitsZo** [comment]
About the git commit message, you can talk with me at #16 .

**Comment by RileyYe** [comment]
👀

**Review by TTL2000 — APPROVED** [review]

**Review by TTL2000 — APPROVED** [review]

**Review by PeterlitsZo — APPROVED** [review]
Good work

**Pull request merged**
